deployment/environment/memory/environment.go

```diff
@@ -29,6 +29,18 @@ type MemoryEnvironmentConfig struct {
 	RegistryConfig deployment.CapabilityRegistryConfig
 }
 
+// For placeholders like aptos
+func NewMemoryChain(t *testing.T, selector uint64) deployment.Chain {
```

Comment:
Note TODO here regarding the thinking for non-evms https://github.com/smartcontractkit/chainlink/blob/c6c8fbd559091bd3fb6673c7fb20c90a92aebe1d/deployment/environment.go#L63. Fine if we intend to come back to it, but intention is we'd add some aptos go sdk based Chain there (will require bumping geth to 1.14, which I'm working on with @jmank88)

Reply:
I ended up not even using this in the tests since chain-selectors doesn't support non-EVMs, but most other chains don't have a simulated backend. We'd need to start a local node and test against that. For Keystone tests at the moment we don't actually need a fully configured node, just an instance of the node with the Aptos keys available so we can configure the contracts.

Reply:
chain-selectors fully was intended to support non-evms, should be fixed soon. 

>  but most other chains don't have a simulated backend. We'd need to start a local node and test against that.

Yep the intention there is to use the docker based environment. The AptosOnchainClient or w/e interface would just have an implementation only in the devenv not memory.